.drone.yml

```diff
@@ -176,7 +176,6 @@ pipeline:
   report-coverage:
     image: plugins/coverage
     server: https://coverage.vmware.run
-    token: ${GITHUB_AUTOMATION_API_KEY}
```

Comment:
The plugin accepts a secret env var `GITHUB_TOKEN` from drone, it is still being passed in, not via the yml but via drone .
As per their docs https://github.com/drone-plugins/drone-coverage